components/expenses/ExpenseGSTFormikFields.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TaxType } from '@opencollective/taxes';
+import { isNil } from 'lodash';
+import { useIntl } from 'react-intl';
+
+import { createError, ERROR } from '../../lib/errors';
+import { verifyValueInRange } from '../../lib/form-utils';
+
+import { Grid } from '../Grid';
+import StyledInput from '../StyledInput';
+import StyledInputFormikField from '../StyledInputFormikField';
+import StyledSelect from '../StyledSelect';
+
+export const validateTaxGST = (intl, tax) => {
```

Comment:
This function could be in `@opencollective/taxes`.

Reply:
Since the last iteration, the rate can't be customized. The only validation is for the `idNumber` field to be set. It's closely tied to formik state and intl, so I'll keep it in there